src/axom/sidre/spio/interface/c_fortran/wrapIOManager.cpp

```diff
@@ -18,86 +18,89 @@ extern "C" {
 // splicer begin class.IOManager.C_definitions
 // splicer end class.IOManager.C_definitions
 
```

Comment:
Somehow clangformat handled the `*` differently in this file than in any other file.  I don't know why this is.